lib/lte_link_control/lte_lc.c

```diff
@@ -330,10 +653,27 @@ int lte_lc_init(void)
 	return w_lte_lc_init();
 }
 
+int lte_lc_register_handler(lte_lc_evt_handler_t handler)
+{
+	if (handler == NULL) {
+		LOG_ERR("Handler cannot be NULL");
+		return -EINVAL;
+	}
+
+	if (evt_handler) {
+		LOG_WRN("Replacing previously registered handler (%p) with %p",
+			evt_handler, handler);
+	}
+
+	evt_handler = handler;
```

Comment:
I think we should be able to unregister a handler.

Reply:
Agreed, allowing NULL now to unregister